modules/mod_ginger_rdf/models/m_rdf_export.erl

```diff
@@ -23,7 +23,10 @@ to_rdf(Id, Context) ->
 %%      - media.
 -spec to_rdf(m_rsc:resource(), [module()], z:context()) -> m_rdf:rdf_resource().
 to_rdf(Id, Ontologies, Context) ->
-    Properties = m_rsc:get_visible(Id, Context),
+    Properties = case m_rsc:get_visible(Id, Context) of
+        undefined -> [];
+        Props -> Props
+    end,
```

Comment:
Just for the record, from our slack conversation: This rdf change was snucked in, unrelated to template render controller. Maybe we need to apply this in a separate pull request; maybe it has been lying around for too long because it potentially fixes bugs?

Reply:
It potentially fixes bugs.

Ideally it should have been a separate merge request.